image/config.go

```diff
@@ -68,6 +66,25 @@ func findConfig(w walker, d *descriptor) (*config, error) {
 	}
 }
 
+func (c *config) validate(w walker) error {
+
+	// check if the rootfs type is 'layers'
+	if c.RootFS.Type != "layers" {
+		return errors.New("RootFs is an unknown rootfs type, MUST be 'layers'")
+	}
```

Comment:
We don't need to check this in Go, because it's checked in JSON Schema since opencontainers/image-spec#358.